addons/mail/models/mail_composer_mixin.py

```diff
@@ -22,7 +22,7 @@ class MailComposerMixin(models.AbstractModel):
 
     # Content
     subject = fields.Char('Subject', compute='_compute_subject', readonly=False, store=True)
-    body = fields.Html('Contents', sanitize_style=True, compute='_compute_body', store=True, readonly=False)
+    body = fields.Html('Contents', compute='_compute_body', render_engine='qweb', store=True, readonly=False, sanitize=False)
```

Comment:
Well did you test all use cases ? Sanitizing things is not only a security matter (well, it is), but alos preventing people from breaking other people displays (either by mistake or when wanting it). Notably styling allowing images to take the whole screen, phishing, ... 

cc @std-odoo